net/iperf3/files/etc/init.d/iperf3

```diff
@@ -0,0 +1,121 @@
+#!/bin/sh /etc/rc.common
+
+USE_PROCD=1
+
+START=50
+PROG_DEFAULT="/usr/bin/iperf3"
+PROG_REDIRECT="/usr/libexec/iperf3-ubus-redirect"
+
+DIR="/var/run/iperf3"
+PIDFILE="${DIR}/pidfile"
+LOGFILE="${DIR}/logfile"
+
+. $IPKG_INSTROOT/lib/functions/network.sh
+
+start_server() {
+	local cfg="$1"
+
+	local enabled port iface interval proto ip redirect prog
+
+	config_get_bool enabled "$cfg" 'enabled' 1
+	[ $enabled -gt 0 ] || return
+
+	rm -f "${LOGFILE}-${cfg}.log"
+
+	config_get port "$cfg" port '5201'
```

Comment:
If there's a chance people would be defining multiple servers, would it maybe be better to implement lazy port specification, where you set the default value in [start_service](https://github.com/openwrt/packages/blob/master/net/https-dns-proxy/files/https-dns-proxy.init#L155) and then [use it](https://github.com/openwrt/packages/blob/master/net/https-dns-proxy/files/https-dns-proxy.init#L91) and [increment it](https://github.com/openwrt/packages/blob/master/net/https-dns-proxy/files/https-dns-proxy.init#L149) in the instance start.

Reply:
good advice. I have added it.
https://github.com/openwrt/packages/pull/16282/commits/c42bade1bbaef53700d540ce5127551d4473cbdf

Will stash the commits if it works

Reply:
If you could help me understand why you want back to static port number from the lazy/dynamic port number, I'd be grateful.

Reply:
I have now decided against the lazy port binding. When I have a server, I want to specify exactly which port it should listen to.